cogs/commandlistener.py

```diff
@@ -50,6 +50,8 @@ async def on_ready(self):
         await self.utils.sendModsMessage(
             "Bot is now online.\nVersion:\tDiscordBot DataBot v2.2.1"
         )
+        # Sets the bot's presence to "Do not Disturb" to indicate it's not logging.
+        await self.bot.change_presence(status=discord.Status.dnd, activity=discord.Game(str(self.jh.getFromConfig("command_prefix")) + "help"))
```

Comment:
In the config the login state can be configured. Thus, the bot can log from the start.